src/pages/profile/console.ts

```diff
@@ -110,6 +110,73 @@ class ConsoleComponent implements m.ClassComponent<ConsoleAttributes> {
     this.autoScroll = isAtBottom
   }
 
+  parseLogEntry(entry: string) : m.Vnode {
+    const colorizeDid = (entry: string): m.Vnode => {
+      const didRegex = /(.*)(did):([a-z0-9]+):((?:[a-zA-Z0-9._%-]*:)*[a-zA-Z0-9._%-]+)([?/#][a-zA-Z0-9._%-])?(".*)/
+      if(!didRegex.test(entry))
+        return m("span", entry);
+      const match = didRegex.exec(entry);
+      return m(
+        "span",
+        match[1],
+        m("span", {style:{color:"#d41f0b"}}, match[2]),
```

Comment:
I think it would be preferable to assign a class so all the colors can be in the css with easy to parse names